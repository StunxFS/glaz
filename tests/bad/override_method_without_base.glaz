struct St {
}

impl St {
    @[override]
    fn x(&self) {}
}

fn main() {}
