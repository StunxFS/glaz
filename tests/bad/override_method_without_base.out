tests/bad/override_method_without_base.glaz:6:9: error: `x` cannot be marked as override
     | 
   6 |     fn x(&self) {}
     |        ^^^^^^^^
     = note: struct `St` has no base
glaz error: could not compile `override_method_without_base`, aborting due to previous error.
